### ISSUE TITLE
Decorative image is not defined under decorative.

### ISSUE THREAD

**Issue by 232amansingh** [microsoft/BotFramework-WebChat#5747]
### Is it an issue related to Adaptive Cards?

Yes, this is related to Adaptive Card. However, it does not repro on Adaptive Cards Designer.

### What is the PWD impact?

When a decorative image is not defined as decorative, screen reader users may hear unnecessary announcements like “image” or “graphic” without meaningful context. This adds noise, distracts from important information, and slows navigation.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

Repro Steps: ​           
1. Open URL: Web Chat in edge browser.
2.Ask query 'Carousel'.
3. Navigate to the cards in browse mode.
4. Observe the issue that Decorative image is not defined under decorative. 

### What do you expect?

Decorative image should be defined as decorative and in scan mode, screen reader focus should not move to the decorative images.

### What actually happened?

Decorative image is not defined under decorative. 

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/4bd1d022-7248-42bd-af16-13122b689f44

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/non-text-content

### Adaptive Card JSON

```json

```
